Formula/bubblewrap.rb

```diff
@@ -0,0 +1,61 @@
+class Bubblewrap < Formula
+  desc "Unprivileged sandboxing tool for Linux"
+  homepage "https://github.com/containers/bubblewrap"
+  url "https://github.com/containers/bubblewrap/releases/download/v0.4.1/bubblewrap-0.4.1.tar.xz"
+  sha256 "b9c69b9b1c61a608f34325c8e1a495229bacf6e4a07cbb0c80cf7a814d7ccc03"
+  license "LGPL-2.0-or-later"
+
+  head do
+    url "https://github.com/containers/bubblewrap.git", branch: "master"
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+  end
+
+  depends_on "docbook-xsl" => :build
+  depends_on "libxslt"     => :build
+  depends_on "libcap"
+  depends_on :linux
+
+  def install
+    system "autoreconf", "-fvi" if build.head?
+    system "./configure", *std_configure_args, "--disable-silent-rules",
+           "--with-bash-completion-dir=#{bash_completion}"
+
+    inreplace "Makefile" do |s|
+      s.gsub! "http://docbook.sourceforge.net/release/xsl/current",
+              "#{Formula["docbook-xsl"].opt_prefix}/docbook-xsl"
+    end
```

Comment:
Why would we replace this?

Reply:
The Makefile passes the `HTTP URI` to `xsltproc`, which then tries to fetch this file over `HTTP`, which is not possible inside the build on the github runner, but the local file is provided by "depends_on "docbook-xsl"', so xsltproc then uses the local file.